qutebrowser/config/configtypes.py

```diff
@@ -1151,6 +1162,17 @@ class UserStyleSheet(File):
     def __init__(self):
         super().__init__(none_ok=True)
 
+    def transform(self, value):
+        if not value:
+            return None
+        path = super().transform(value)
+        path = os.path.expandvars(path)
```

Comment:
I initially put both expanduser and expandvars in File.transform and removed them from UserStyleSheet.transform, but that triggered
```FAIL tests/config/test_configtypes.py::TestFile::()::test_transform```
and I couldn't get my head around why. So I left expandvars in UserStyleSheet.transform for now. I must admit that the test cases are still somewhat over my head.